.templates/new-module/tests/react/webpack.config.js

```diff
@@ -0,0 +1,15 @@
+module.exports = {
+	entry: './index.jsx',
+	output: {
+		filename: './bundle.js'
+	},
+	module: {
+		rules: [
+			{
+				test: /\.jsx$/,
```

Comment:
I think we could improve this. I worry that the domain logic requires any react file to have the `.jsx` extension. I'm sure this will cause bugs or confusion in the future with adding modules. Personally I don't use the jsx extension.